db/migrate/20170414171825_change_timestamps_to_utc.rb

```diff
@@ -0,0 +1,78 @@
+class ChangeTimestampsToUtc < ActiveRecord::Migration
+  TABLES = {
+    audits: [:created_at],
+    board_authors: [:created_at, :updated_at],
+    board_sections: [:created_at, :updated_at],
+    board_views: [:created_at, :updated_at, :read_at],
+    boards: [:created_at, :updated_at],
+    character_aliases: [:created_at, :updated_at],
+    character_tags: [:created_at, :updated_at],
+    characters: [:created_at, :updated_at],
+    favorites: [:created_at, :updated_at],
+    flat_posts: [:created_at, :updated_at],
+    galleries: [:created_at, :updated_at],
+    icons: [:created_at, :updated_at],
+    messages: [:read_at, :created_at, :updated_at],
+    password_resets: [:created_at, :updated_at],
+    post_tags: [:created_at, :updated_at],
+    post_viewers: [:created_at, :updated_at],
+    post_views: [:created_at, :updated_at, :read_at],
+    posts: [:created_at, :updated_at, :edited_at, :tagged_at],
+    replies: [:created_at, :updated_at],
+    reply_drafts: [:created_at, :updated_at],
+    tags: [:created_at, :updated_at],
+    templates: [:created_at, :updated_at],
+    users: [:created_at, :updated_at]
+  }
+
+  def fetch_zone
+    zone_name = Time.now.zone # GMT, BST, etc.
+    # this list is likely overkill
+    zone_maps = {
+      "UTC" => ["UTC"],
+      "Europe/London" => ["GMT", "BST"],
+      "America/New_York" => ["EST", "EDT"],
+      "America/Chicago" => ["CST", "CDT"],
+      "America/Denver" => ["MST", "MDT"], # sorry Phoenix
+      "America/Los_Angeles" => ["PST", "PDT"],
+      "Europe/Paris" => ["CET", "CEST"], # there is no general Europe one?
+      "Europe/Lisbon" => ["WET", "WEST"],
+      "America/Sao_Paulo" => ["-02", "-03"] # may get false positives, but we don't have many Brazilians
+      # no Australians etc probably?
+      # none who are actively developing at least and
+      # this migration will become outdated but unnecessary
+    }
+    zone_maps.each do |olson, zones|
+      return olson if zones.include?(zone_name)
+    end
+    raise("Failed to map timezone to region.")
+  end
+
+  def up
+    local_zone = fetch_zone
```

Comment:
Yes. Or local servers (development servers), as this migration will affect those. (It's not much of a problem if local times get to be a few hours out of sync, I don't think, but seems preferable to try to not.)